api/src/main/java/uk/gov/dhsc/htbhf/claimant/entitlement/CycleVoucherEntitlement.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.dhsc.htbhf.claimant.entitlement;
+
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * Represents the number (and value) of vouchers a claimant is entitled to for a given payment cycle.
+ */
+@Data
+public class CycleVoucherEntitlement {
+
+    private final int vouchersForChildrenUnderOne;
+    private final int vouchersForChildrenBetweenOneAndFour;
+    private final int vouchersForPregnancy;
+    private final int totalVoucherEntitlement;
+    private final int voucherValueInPence;
+    private final int totalVoucherValueInPence;
+
+    public CycleVoucherEntitlement(List<VoucherEntitlement> voucherEntitlements) {
+        if (voucherEntitlements == null || voucherEntitlements.isEmpty()) {
```

Comment:
How about `CollectionUtils.isEmpty()`?

Reply:
Didn't realise that also checked for null, updated.